File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int input;
    do
    {
        // prompts user for pyramid height between 1 and 8
        input = get_int("Height: ");
    }
    while (input < 1 || input > 8);

    if (input > 0 || input < 9)
    {
        int counter = 0;
        // for loops specifying number of spaces and hashes per line
        for (int i = 0; i < input; i++)
        {
            if (counter != input)
            {
                //for loop to print out the spaces
                for (int j = (input - 1) - counter; j > 0; j--)
                {
                    printf(" ");
                }
                //for loop to print out the hashes
                for (int k = 0; k <= counter; k++)
                {
                    printf("#");
                }
                printf("\n");
                counter++;
            }
        }
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int


while True:
    # Get the input from the user
    height = get_int("Height : ")
    # check the condition between 1 and 8
    if height >= 1 and height <= 8:
        break

# Loop over the rows
for rows in range(0, height):
    # To print the spaces
    for spaces in range(1, height - rows):
        print(" ", end="")

    # To print the hashes
    for rightHashes in range(height - rows, height + 1):
        print("#", end="")

    print()